src/applications/vre/28-8832/config/form.js

```diff
@@ -25,47 +32,57 @@ const formConfig = {
   title: '28-8832-planning-and-guidance',
   defaultDefinitions: {},
   chapters: {
-    veteranDetails: {
-      title: 'Personal Information',
-      pages: {
-        veteranInformation: {
-          path: 'first-name',
-          title: 'Personal Information - Page 1',
-          uiSchema: veteranInformation.uiSchema,
-          schema: veteranInformation.schema,
-        },
-      },
-    },
-    militaryHistory: {
-      title: 'MIlitary History',
+    statusSelection: {
+      title: 'Your Status',
       pages: {
-        militaryHistory: {
-          path: 'military-history',
-          title: 'Military History',
-          uiSchema: militaryHistory.uiSchema,
-          schema: militaryHistory.schema,
+        wizard: {
```

Comment:
It might be worthwhile renaming this for the sake of clarity, since the designs call for the use of an actual wizard pattern prior to the introduction page. Just an opinion though.

Reply:
Was this renamed? I refreshed my browser but don't see an update here.